Resfeber/Login/LoginViewController.swift

```diff
@@ -36,6 +40,32 @@ class LoginViewController: UIViewController {
     
     // MARK: - Private Methods
     
+    fileprivate func setupView() {
+        view.backgroundColor = UIColor.Resfeber.background
+        
+        // Sets logoImageView to render as a template image and sets the color to ResfeberRed
+        logoImageView.image = logoImageView.image?.withRenderingMode(.alwaysTemplate)
+        logoImageView.tintColor = UIColor.Resfeber.red
+        
+        // Sets edge insets for button text
+        signInButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: 5, bottom: 0, right: 5)
+        
+        signInButton.layer.cornerRadius = 5
+        signInButton.layer.shadowOffset = CGSize(width: 0, height: 1.0)
+        signInButton.layer.shadowOpacity = 0.4
+        signInButton.layer.shadowRadius = 5
+        signInButton.layer.shouldRasterize = true
+        signInButton.layer.masksToBounds =  false
+        
+        // Only displays shadow on button in light mode
+        if traitCollection.userInterfaceStyle == .dark {
+            signInButton.layer.shadowColor = nil
+        } else {
+            signInButton.layer.shadowColor = UIColor.Resfeber.dark.cgColor
+        }
```

Comment:
Something to think about is how is this going to look if the user changes to dark mode or light mode after the view has already loaded. Maybe you won't even see the shadow because the other elements around it will be dark? Is the shadow going to look weird? You could put this into it's own method and call that method when the trait collection changes.

Reply:
Removed code. Will work with UX to finalize their requirements- if shadow is required then will put in own method